src/coreclr/src/tools/Common/Compiler/DependencyAnalysis/SettableReadOnlyDataBlob.cs

```diff
@@ -12,13 +12,13 @@ namespace ILCompiler.DependencyAnalysis
 {
     public class SettableReadOnlyDataBlob : ObjectNode, ISymbolDefinitionNode
     {
-        private Utf8String _name;
+        private MethodDesc _owningMethod;
         private ObjectNodeSection _section;
         private ObjectData _data;
 
-        public SettableReadOnlyDataBlob(Utf8String name, ObjectNodeSection section)
+        public SettableReadOnlyDataBlob(MethodDesc owningMethod, ObjectNodeSection section)
```

Comment:
Since this is making the node not general purpose anymore, maybe we can go all the way and remove the `section` parameter from the constructor and name the node in a way that conveys its purpose (`MethodReadOnlyDataNode`?).

Reply:
Good idea - done

Reply:
Could you rename the file as well? :)

Reply:
Ugh, VS crashed during refactor. Will do.